OpenRA.Mods.Common/Traits/Render/WithCrateBody.cs

```diff
@@ -23,6 +23,10 @@ class WithCrateBodyInfo : ITraitInfo, Requires<RenderSpritesInfo>
 		[Desc("Easteregg sequences to use in december.")]
 		public readonly string[] XmasImages = { };
 
+		[SequenceReference] public readonly string IdleSequence = "idle";
+		[SequenceReference] public readonly string WaterSequence;
+		[SequenceReference] public readonly string LandSequence;
```

Comment:
Convention is to explicitly specify null in these cases, even though they aren't needed.